bin/cli.js

```diff
@@ -272,6 +277,12 @@ const runBrowser = (config, cb) => {
   const headers = [];
   headers.push('__karma__.start=()=>{}');
   headers.push('require(\'babel-polyfill\')');
+  if (config.reporter === 'tap') {
+    headers.push(
```

Comment:
Since `karma` cann't tell whether tests are running for `metatests` themselves or for some other package, headers should be changed based on this similar to [`runTodo`](https://github.com/metarhia/metatests/pull/135/files#diff-22d5d2b57c69d2e1ac054c4d0f744ff1R286).

Reply:
`if (require('../package.json').name === 'metatests') {
      headers.push('require(\'..\').runner.instance.TapReporter()');
    } else {
      headers.push('require(\'metatests\').runner.instance.TapReporter()');
    }` 
Should I write this way?

Reply:
Yes, this is correct. Also, you can do it like this:
```js
const metatestsRequire = require('../package.json').name === 'metatests' ?
  'require(\'..\')' :
  'require(\'metatests\')';
headers.push(`${metatestsRequire}.runner.instance.TapReporter()`);
```
so it will be easier to add `metatests` specific options like `reporter` or `runTodo`.

Reply:
Actually, it should be
```js
headers.push(
  `${metatestsRequire}.runner.instance.setReporter(` +
  `new ${metatestsRequire}.reporters.TapReporter())`
);
```
Sorry for misleading, I didn't notice there were no `setReporter`.